app/src/main/java/com/infomaniak/mail/ui/main/search/SearchFragment.kt

```diff
@@ -131,8 +132,10 @@ class SearchFragment : TwoPaneFragment() {
 
     private fun handleEdgeToEdge(): Unit = with(binding) {
         applyWindowInsetsListener(shouldConsume = false) { _, insets ->
-            appBar.applyStatusBarInsets(insets)
+            toolbar.applySideAndBottomSystemInsets(insets, withBottom = false)
+            toolbar.setMargins(top = insets.safeArea().top) // Apply margins to keep the icon and the search bar aligned
```

Comment:
The comment needs to explain more clearly why suddenly we can't just use paddings like everywhere else and need to use margin and split it from the horizontal padding from the line above it.

We can easily see that the line that calls `setMargins()` is used to ... set margin, don't worry 😂